app/domain/authentication/authn_jwt/extract_token_from_credentials.rb

```diff
@@ -0,0 +1,31 @@
+module Authentication
+  module AuthnJwt
+
+    ExtractTokenFromCredentials ||= CommandClass.new(
+      dependencies: {
+        decoded_credentials_class: Authentication::Jwt::DecodedCredentials
+      },
+      inputs: %i[credentials]
+    ) do
+
+      def call
+        decode_credentials
+        extract_token_from_credentials
+      end
+
+      private
+
+      def decode_credentials
+        decoded_credentials
+      end
+
+      def decoded_credentials
+        @decoded_credentials ||= @decoded_credentials_class.new(@credentials)
+      end
+
+      def extract_token_from_credentials
+        decoded_credentials.jwt
+      end
```

Comment:
sorry, i would add also:
```suggestion
      def extract_token_from_credentials
        extracted_token
      end
      
      def extracted_token
        @extracted_token ||= decoded_credentials.jwt
      end
```

and add `extracted_token` to the end of the `call` function. this way the call function will be:
```
decode_credentials
extract_token_from_credentials
extracted_token
```

which tells the story best